pkg/volume/gce_pd/attacher_test.go

```diff
@@ -342,6 +342,6 @@ func (testcase *testcase) DeleteDisk(diskToDelete string) error {
 	return errors.New("Not implemented")
 }
 
-func (testcase *testcase) GetAutoLabelsForPD(name string) (map[string]string, error) {
+func (testcase *testcase) GetAutoLabelsForPD(name string, zone string) (map[string]string, error) {
```

Comment:
I guess we could have two tests: one that tests that Petsets creates PVCs with names like `<prefix>-<ordinal>`, and the other that checks that PVCs with names like `<prefix>-<ordinal>` are round-robin-ed across zones.